From khali@linux-fr.org Wed Jul  5 09:05:22 2006
Date: Wed, 5 Jul 2006 18:05:15 +0200
From: Charles Spirakis <bezaur@gmail.com>
To: Greg KH <greg@kroah.com>
Cc: LM Sensors <lm-sensors@lm-sensors.org>, Charles Spirakis <bezaur@gmail.com>
Subject: hwmon: New maintainer for w83791d
Message-Id: <20060705180515.97a0691d.khali@linux-fr.org>
Content-Disposition: inline; filename=hwmon-w83791d-maintainer.patch

From: Charles Spirakis <bezaur@gmail.com>

Add Charles Spirakis as the maintainer for the w83791d sensor chip
driver.

Signed-off-by: Charles Spirakis <bezaur@gmail.com>
Signed-off-by: Jean Delvare <khali@linux-fr.org>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>

---
 MAINTAINERS |    6 ++++++
 1 file changed, 6 insertions(+)

--- gregkh-2.6.orig/MAINTAINERS
+++ gregkh-2.6/MAINTAINERS
@@ -3182,6 +3182,12 @@ P:	Evgeniy Polyakov
 M:	johnpol@2ka.mipt.ru
 S:	Maintained
 
+W83791D HARDWARE MONITORING DRIVER
+P:	Charles Spirakis
+M:	bezaur@gmail.com
+L:	lm-sensors@lm-sensors.org
+S:	Maintained
+
 W83L51xD SD/MMC CARD INTERFACE DRIVER
 P:	Pierre Ossman
 M:	drzeus-wbsd@drzeus.cx
